nistats/tests/unit_tests/test_verify_events_file_uses_tab_separators.py

```diff
@@ -0,0 +1,147 @@
+import csv
```

Comment:
should be  nistats/tests/test_verify_events_file_uses_tab_separators.py
i don't see the usefulness of adding the unit_tests levels as all our tests are unit_tests anyhow.

Reply:
I have no problem in putting them in tests.

Reply:
There's also the idea that I will be adding more unit tests as time goes on, and the current tests are more integration tests than unit tests- they don't test individual functions or methods but small pipeline of multiple functions.
I will defer to your opinion here. Let me know.

Reply:
Maybe, but for the moment I'd like to keep the thing homogeneous (and homogeneous with Nilearn). So let's keep that on a TODO list.